README.md

```diff
@@ -34,7 +34,7 @@ Access design documentation specific to the KNI and KubeVirt features in OpenShi
 ### KubeVirt (Virtualization in OpenShift)
 - Create VM
 - Clone VM
-- Migrate VM
+- [Migrate VM](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/migrate-vm/migrate-vm)
```

Comment:
one little thing, we're going to drop the absolute paths and do relative instead. So follow this format:
`[Migrate VM](web-console/knikubevirt/migrate-vm/migrate-vm)`

Reply:
With a `.md` at the end too, so:

`[Migrate VM](web-console/knikubevirt/migrate-vm/migrate-vm.md)`

Reply:
My fault on this one...but after some discussion with @matthewcarleton and @andybraren, we currently still have absolute paths. Sorry for the headfake @yfrimanm